bundle.md

```diff
@@ -8,7 +8,7 @@ See also [OS X application bundles](http://en.wikipedia.org/wiki/Bundle_%28OS_X%
 The definition of a bundle is only concerned with how a container, and its configuration data, are stored on a local file system so that it can be consumed by a compliant runtime.
 
 A Standard Container bundle contains all the information needed to load and run a container.
-This includes the following artifacts which MUST all reside in the same directory on the local filesystem:
+This MUST include the following artifacts:
```

Comment:
I agree with remove the bit about "same dir" from above, but then I think you need to add it below for each artifact that still need to be in that dir - like config.json.